### PR TITLE
*DO NOT MERGE* Draft PR

### LINKED ISSUE #5: Add APIs related to Users and Studios
**Describe the bug**
We need an API which connects users to skills and role and connects studio to users both at macroservice and aggregator level.

**Expected behavior**
Feign Clients should pick up the API connected to the aggregator

**Additional context**
- Make sure to have appropriate test cases(both unit and integration, also on the server) to confirm the working as well.

### PR DESCRIPTION
**Pull request by picksitquick**
Fixes #5 

## Proposed Changes

  - Adds Roles to User
  - Adds Skills to User
  - Adds Studio to Users
  
Need a draft review before we can take the next steps, Existing Error handlers have been used, if needed new custom error handlers can be made and used.

### CODE REVIEW

**Comment by aprajshekhar** [comment]
Thanks for the PR.

Few suggestions:

1. Please try to avoid single statement if clauses.
2. Create a new exception for validation error.

File: pom.xml
Comment (aprajshekhar) [inline pom.xml:1]: The diff is not displaying the changes completely. The distribution management a.k.a maven publish section is not there in the pom.xml of the PR.

**Review by aprajshekhar — CHANGES_REQUESTED** [review]

**Comment by aprajshekhar** [comment]
Hi @picksitquick shall I  merge  this PR?

**Comment by picksitquick** [comment]
> Hi @picksitquick shall I merge this PR?

Yes please

**Review by aprajshekhar — APPROVED** [review]

**Pull request merged**